src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -991,6 +991,132 @@ public async Task<Organisation> GetOrganisationForApplication(Guid applicationId
             }
         }
 
+        public async Task<List<ApplicationOversightDetails>> GetOversightsPending()
```

Comment:
@ConorTill You've added these repo methods (back?) into this class, but there don't seem to be any usages. Could you check, and remove if not needed.